fendermint/vm/interpreter/src/fvm/checkpoint.rs

```diff
@@ -116,6 +121,14 @@ where
         power_diff(curr_power_table, next_power_table)
     };
 
+    emit!(
+        VMEvent::NewBottomUpCheckpoint,
+        height = height.value(),
+        block_hash = hex::encode(block_hash),
```

Comment:
Perhaps we add a fmt::Display implementation to whatever type `block_hash` is instead? I see us doing this in several places.

Reply:
block_hash is `Vec<u8>` or slice, unless we crate a wrapper type, it's kind of difficult to impl that.